rib/rib.go

```diff
@@ -188,6 +191,10 @@ type RIBHolder struct {
 	// groups and next-hops within the RIB. It is used to ensure that referenced NHs
 	// and NHGs cnanot be removed from the RIB.
 	refCounts *niRefCounter
+
+	// disableForwardRef indicates that this RIB should not allow
+	// references to entities that do not yet exist.
+	disableForwardRef bool
```

Comment:
We need to propagate this down to the `RIBHolder` because an individual `RIBHolder` (VRF RIB) does not know its part of a parent RIB. We have to store it at the parent level, because that RIB can have new VRF RIBs created within it, and we absolutely want it to be consistent.

Fixed the naming to be consistent (to diable).